src/fsharp/lib.fs

```diff
@@ -564,3 +564,4 @@ type MaybeLazy<'T> =
         | Strict x -> x
         | Lazy x -> x.Force()
 
+let inline vsnd ((_, y): struct('T * 'T)) = y
```

Comment:
why the `('T*'T)` here and not `('a*'b)` ?

Reply:
Agreed, this should be `('a * 'b)` or `('T * 'U)`.